src/Scrapers/Scraper.py

```diff
@@ -12,13 +16,26 @@ class Scraper:
     def __init__(self, url) -> None:
         self.url = url
 
-    def __getResponse(self):
+    def __getResponse(self, isDynamic=False):
         # Because request only makes http request calls, change our https link to http
-        protocol , link = self.url.split(':')
-        protocol = 'http'
-        url = protocol + ':'+link
+        protocol, link = self.url.split(':')
+        if not isDynamic:
+            protocol = 'http'
+        else:
+            protocol = 'https'
+        url = protocol + ':' + link
         proxy = Config().get_proxy()
-        response = requests.get(url,proxies = proxy,verify = False)
+        if not isDynamic:
+            response = requests.get(url,proxies = proxy,verify = False)
+        else:
+            os.environ["PATH"] += os.pathsep + os.getcwd().split("src",1)[0]
+            options = Options()
+            options.headless = True
+            driver = webdriver.Firefox(options=options)
+            driver.get(url)
+            time.sleep(5)
+            response = driver.page_source
+            
```

Comment:
We can modularize above parts by creating separate private handlers for both static and dynamic content.

Also, we can think about overriding this method in CCScraper.

Reply:
If I am not wrong, do you mean I should group the steps for handling dynamic websites together into some private method and then call this method wherever needed? Should I define this method inside Scraper.py itself or create a new module?

Reply:
Yes, we can group the steps for handling dynamic and static websites.

About your 2nd question, what the method does obviously does not change but how it is done can change. Nevertheless, you can go with the 1st option.

Reply:
Based on the suggestions I have made the following changes @akstron :
- Changed camel case to snake case
- Added the UnsupportedPlatform exception
- Created separate private handlers for handling static and dynamic websites